src/webgl/3d_primitives.js

```diff
@@ -1588,13 +1588,62 @@ p5.RendererGL.prototype.bezierVertex = function(...args) {
 
     const LUTLength = this._lookUpTableBezier.length;
 
+    // Get the last fill color
+    const _vertexColors = this.immediateMode.geometry.vertexColors;
+    const _vertexColorSize = _vertexColors.length;
+    const lastFillColor = [
+      _vertexColors[_vertexColorSize - 4],
+      _vertexColors[_vertexColorSize - 3],
+      _vertexColors[_vertexColorSize - 2],
+      _vertexColors[_vertexColorSize - 1]
+    ];
+    const finalFillColor = this.curFillColor.slice();
+
     if (argLength === 6) {
       this.isBezier = true;
 
       w_x = [this.immediateMode._bezierVertex[0], args[0], args[2], args[4]];
       w_y = [this.immediateMode._bezierVertex[1], args[1], args[3], args[5]];
+      // calCalculate intermediate colors
+      let d0 = Math.sqrt(
```

Comment:
We can use [`Math.hypot()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot) here instead of sqrt of sums of pows to be a little more concise

Reply:
I didn't know Math.hypot existed...I think I'll use it because it's convenient!